src/tracking/tracking.ts

```diff
@@ -127,21 +127,26 @@ export function recordObservation(dependency: TrackedObservable) {
 
 export interface Observable {
     readonly id: number;
+    /** @internal */
     version: number;
 }
 
 export interface TrackedObservable extends Observable {
+    /** @internal */
     readonly observers: Observer[];
 }
 
 export interface Observer {
+    /** @internal */
     disconnect(): void;
     mark(reactorSink: Reactor[]): void;
```

Comment:
I believe `mark` should be internal as well. Can you check why it isn't already?